catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/security/AccessControlAccessPluginTest.java

```diff
@@ -137,6 +138,62 @@ public void testBasicUpdateFailsWithMissingAdmin() throws Exception {
     accessPlugin.processPreUpdate(update, ImmutableMap.of(MOCK_METACARD_ID, before));
   }
 
+  @Test
+  public void testSuccessWhenWriteableUserModifies() throws StopProcessingException {
+    Metacard before =
+        AccessControlUtil.metacardFromAttributes(
+            ImmutableMap.of(
+                Core.ID,
+                MOCK_METACARD_ID,
+                Core.METACARD_OWNER,
+                "before",
+                SecurityAttributes.ACCESS_INDIVIDUALS,
+                ImmutableList.of(MOCK_IDENTITY_ATTR)));
+
+    Metacard after =
+        AccessControlUtil.metacardFromAttributes(
+            ImmutableMap.of(
+                Core.ID,
+                MOCK_METACARD_ID,
+                Core.METACARD_OWNER,
+                "before",
```

Comment:
✏️ Should this be "after"? I know this doesn't really change anything logically, just wondering if it's a typo

Reply:
nah, I'm not changing the owner.